_ont/ont-cig-g-97cp.md

```diff
@@ -12,17 +12,17 @@ parent: CIG
 | Vendor/Brand     | CIG              |
 | Model            | G-97CP           |
 | ODM              | ✅               |
-| Chipset          | Realtek RTL9601D |
+| Chipset          | Realtek RTL8672 |
 | Flash            |                  |
 | RAM              |                  |
 | System           | Linux            |
 | 2.5GbaseT        | Yes              |
 | Optics           | SC/APC           |
-| IP address       | 192.168.1.1      |
+| IP address       | 192.168.100.1      |
 | Web Gui          |                  |
-| SSH              |                  |
+| SSH              |✅ user root, password the one generated with [ZTE F6005](/ont-zte-f6005)                 |
```

Comment:
Please format the entire table so that the | are vertically aligned (plugins exist for any ide)

Reply:
Done